toolchains/cpu.spack.gyselalibxx_env/gyselalibxx-env-1.1.0.yaml

```diff
@@ -26,18 +26,32 @@ spack:
     - 'py-matplotlib@3.9'
     - 'py-netcdf4@1.7 +mpi'
     - 'py-numpy@2.1'
+    - 'py-pyyaml@6.0'
     - 'py-scipy@1.14'
     - 'py-sympy@1.13'
     - 'py-xarray@2024.7'
   specs:
-  - lapack
   - matrix:
     - [$packages]
-    - [$%compilers]
   - matrix:
     - [$python-packages]
-    - [$%compilers]
+  repos:
+    patches:
+      git: https://github.com/gyselax/spack.git
+      branch: releases/v2025.11
+    builtin:
+      branch: releases/v2025.11
   view: true
   packages:
     all:
-      variants: ~mpi ~openmp cxxstd=17
+      variants: ~fortran ~mpi ~openmp cxxstd=17
+    blas:
+      require: openblas@0.3
+    lapack:
+      require: openblas@0.3
+    mpi:
+      require: openmpi@5.0
+    pdi:
+      require: ~python
+    py-pandas:
+      require: ~performance
```

Comment:
What decides what is specified here rather than above? The positive requirements (e.g. `+mpi`) are specified above but the negative requirements are here. Is there a more fundamental difference between these kinds of requirements?